data/actions/scripts/other/die.lua

```diff
@@ -1,15 +0,0 @@
-function onUse(player, item, fromPosition, target, toPosition, isHotkey)
```

Comment:
Why did you remove that script?

Reply:
It's was not die.lua, it's dice.lua, I forgot to rename it and deleted, but I added the dice.lua again